src/ListForm.js

```diff
@@ -34,6 +34,23 @@ class ListForm {
     return this.set(this.items.length, item);
   }
 
+  unshift(item) {
+    const items = shallowCopyArray(this.items);
+    items.push(item);
```

Comment:
Why not use `unshift` on this array? That would make the following loop obsolete or not?

Reply:
🤦‍♂️